runtime/onert/core/src/exec/LinearExecutor.cc

```diff
@@ -51,8 +51,10 @@ void LinearExecutor::executeImpl()
     _subject.notifyJobBegin(this, op_seq, backend);
 
     auto &fn_seq = code.fn_seq;
-    bool handle_dynamic_tensor = op_seq->has_dynamic_tensor() || hasDynamicInput();
 
+    fn_seq->initRunning();
```

Comment:
I am going to remove the overhead related to `dynamic_inferer` in fn_sequence like `dynamic_cast`. It is about 3~4% penalty. I would like to move this out of every func sequence run if possible.

Reply:
You're correct. This is a quick fix and should be optimized later.
I will add `todo` into comment mentioning this and keep eye on this.
Thanks.

Reply:
> Please note fn_seq->run() is hot spot. 

I am sorry. I was confused.

initRunning() is not called for every function, but for every function sequence.

`FunctionSequence::run()` has a loop for iterating functions.
`dynamic_cast` is in the loop, and `initRunning` is out of this loop.
Thus, it is not big as I first thought.